src/components/Card/CardFront/styled.ts

```diff
@@ -6,11 +6,16 @@ export const CardWrap = styled.div`
   user-select: none;
 `;
 
-export const Card = styled.div`
+export interface CardStyleProps {
+  $color?: string;
+}
+
+export const Card = styled.div<CardStyleProps>`
   width: 20.375rem;
   height: 27rem;
   border-radius: 1rem;
-  background-color: ${(props) => props.theme.colors["--Primary-blue-500"]};
+  background-color: ${(props) =>
+    props.$color || props.theme.colors["--Primary-blue-500"]};
```

Comment:
```suggestion
  background-color: ${(props) =>
    props.$color ?? props.theme.colors["--Primary-blue-500"]};
```
지금 방식도 문제 없을 것 같지만 null 병합 연산자가 조금 더 안전할 것 같아요~!

Reply:
@wo-o29 ??로 바꾸면 "" 같은 값을 넘겼을 때 기본값이 안 적용되지 않아요?

Reply:
```ts
export interface CardStyleProps {
  $color?: string;
}
```
타입을 옵셔널로 설정해주셔서 `""`같은 문자열을 넣을거면 아예 `$color` 프로퍼티에 값을 안주는게 조금 더 타입이랑 맞지 않을까요??

Reply:
@wo-o29 아 이해했어요.